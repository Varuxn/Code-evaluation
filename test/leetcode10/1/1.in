97 59
8 22
20 85
5 79
0 96
33 37
12 35
19 91
79 95
34 38
4 20
1 74
1 2
35 61
8 25
6 43
38 70
18 23
26 88
27 53
24 76
34 42
4 69
0 1
3 10
2 46
57 66
39 59
8 34
42 90
10 47
3 16
1 6
36 50
27 57
13 17
2 3
4 24
12 48
7 56
5 14
66 84
14 18
67 73
17 71
3 21
11 67
12 32
6 29
43 72
22 55
30 60
13 27
5 28
17 41
0 5
0 87
37 86
2 7
57 63
9 51
17 45
41 77
15 78
48 82
3 4
21 80
2 68
1 9
17 33
6 8
47 64
16 83
10 11
18 36
72 81
41 92
0 15
20 31
10 26
44 62
7 30
5 13
6 40
5 44
11 12
65 94
11 65
12 19
37 49
22 93
31 54
63 89
30 52
24 39
5 58
57 75
