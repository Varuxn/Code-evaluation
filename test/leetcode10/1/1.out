0 1 0 5 0 87 0 96 1 2 1 6 1 74 2 3 2 7 2 46 3 10 3 16 3 21 5 14 5 28 5 79 6 29 6 43 7 56 10 47 14 18 18 23 43 72 79 95 